gstreamer-plugin/git-patch/README.txt

```diff
@@ -0,0 +1,23 @@
+# svt-hevc gstreamer plugin installation
+
+1. Build and install SVT-HEVC 
+- git clone https://github.com/intel/SVT-HEVC
+- cd SVT-HEVC
+- mkdir build && cd build && cmake .. && make -j `nproc` && sudo make install
+
+2. Apply SVT-HEVC plugin and enable libsvthevc to Gstreamer
+- git clone https://gitlab.freedesktop.org/gstreamer/gst-build.git
+- cd gst-build
+- git checkout 1.14
+- meson build
+- cd subprojects/gst-plugins-bad
+- git apply ../SVT-HEVC/gstreamer_plugin/0001-svth265enc-Add-svth265-encoder-element.patch
```

Comment:
new folder: gstreamer-plugin/**git-patch**/

Reply:
I fixed this too.